packages/plugin-ext/package.json

```diff
@@ -20,6 +20,7 @@
     "@theia/task": "^0.5.0",
     "@theia/workspace": "^0.5.0",
     "decompress": "^4.2.0",
+    "getmac": "^1.4.6",
```

Comment:
@marcdumais-work Could you please use your tool to verify this dep?

Reply:
Florent and I did the analysis anyway for the new dependency and its own recursive dependency tree. All licenses found are Foundation-approved: 

![Screenshot from 2019-04-24 10-29-59](https://user-images.githubusercontent.com/25749063/56667522-fb061080-667b-11e9-9801-a0e5611a49a4.png)